metals/src/main/scala/scala/meta/metals/Main.scala

```diff
@@ -35,9 +34,8 @@ object Main {
         .setRemoteInterface(classOf[MetalsLanguageClient])
         .setLocalService(server)
         .create()
-      val underlyingClient = launcher.getRemoteProxy
-      val client = new ConfiguredLanguageClient(underlyingClient, config)(ec)
```

Comment:
What was the motivation for this change?

Reply:
The introduced `UnsupportedDebuggingLspSuite` tests the behavior of the `ConfigurationLanguageClient` which is responsible for suppressing the model-refresh notification. Without this change, we couldn't test it, as the `ConfigurationLanguageClient` was created only inside `Main` object, and then injected to the language server (hence not present in the test suite).

Reply:
I see, makes sense, thanks for the explanation!